wgsl/README.md

```diff
@@ -1,5 +1,15 @@
 # WebGPU Shading Language Specification
 
+## Design Criteria
+
+There are several constraints driving the development of WGSL. The ones related
+to the language itself are found in the [Goals](https://gpuweb.github.io/gpuweb//wgsl#goals)
+section of the specification. Criteria which guide our development of the language are:
+
+ * The burden for adding constructs to the language is to show why the construct
+   is needed, not to show why it should not be added
```

Comment:
I'm having trouble deconstructing this point.  I think I understand it up to the comma (which is what was in #586).  I don't quite get what the part after the comma is trying to add or clarify.

Reply:
Removed. I was trying to be more clear, but failed, heh.